js/src/features/project/components/ProjectDescriptionForm.tsx

```diff
@@ -8,7 +8,7 @@ import { FormErrorMap } from "MyTypes";
 interface Props {
   project: Project;
   edit: (form: EditProjectDescriptionRequest) => void;
-  close: (close: boolean) => void;
+  closeProjectDescriptionForm: () => void;
```

Comment:
Why did you rename it? the name was fine. You are already in component named `ProjectDescriptionForm`. It's obvious that you are closing ProjectDescriptionForm. Why didn't you rename `edit`?